tests/e2e/roles/conftest.py

```
@@ -9,26 +9,29 @@
-    namespace, operatorgroup_name, operatorgroup_yaml
-):
-    # Ensure namespace exists
-    ns_check = subprocess.run(
-        ["oc", "get", "namespace", namespace], capture_output=True, text=True
-    )
-    if ns_check.returncode != 0:
-        subprocess.run(["oc", "create", "namespace", namespace], check=True)
+def ensure_namespace_and_operatorgroup(namespace, operatorgroup_name, operatorgroup_yaml):
+    # Check if namespace exists with better error handling
+    try:
+        ns_check = subprocess.run(
```

Ruff (0.11.9)

Local variable `ns_check` is assigned to but never used

Remove assignment to unused variable `ns_check`

(F841)